src/swap_app_context.h

```diff
@@ -1,10 +1,12 @@
 #ifndef _SWAP_APP_CONTEXT_H_
 #define _SWAP_APP_CONTEXT_H_
 
+#include <os.h>
+#include <cx.h>
```

Comment:
Are those imports needed here?

Reply:
Not `os.h`. `cx.h` is used by module importing this header. I'll clean all these